### ISSUE TITLE
As an admin (on web app) I cannot add a user with a + in the email 

### ISSUE THREAD

**Issue by BastienLQ** [passbolt/passbolt#14]
Hi,

If I want to add a user with a plus in email (ie. `bob+passbolt@example.org`), I cannot validate the form (error "_Only email format is allowed_" is returned). Adding the user from CLI (`app/Console/cake passbolt register_user ...`) works fine.

Thanks.

**Comment by stripthis** [comment]
Hi there,

Do you get this error as a regular user on the registration screen or are you adding the user as an admin? Thanks for the clarification!

**Comment by BastienLQ** [comment]
As an admin.

**Comment by stripthis** [comment]
Thanks @BastienLQ, I could reproduce the issue. We'll get it fixed!
Ref. PASSBOLT-1637
![screen shot 2016-07-25 at 17 31 58](https://cloud.githubusercontent.com/assets/77410/17107096/1a81fe56-528e-11e6-9a8e-967bf27ca588.png)

**Comment by stripthis** [comment]
@BastienLQ this issue has been fixed for the username on the add user screen in release 1.2.0. Please note that we fix issues with '+' not being allowed elswhere like in password description and username in next release. See. issue #17. Thanks!

**Comment by silvaireboivert** [comment]
hello,

the new regex for emails does not match emails with a dot in it eg a.b@c.com 

thanks,

**Comment by kevinmuller** [comment]
Hi @silvaireboivert ,

Indeed, a glitch got introduced in the regexp due to compatibility issues between PHP and Javascript xregexp format. We are working on a quick fix which should be available today itself.

Thanks a lot for the quick feedback!

**Comment by kevinmuller** [comment]
Hi again @silvaireboivert ,

We have just released 1.2.1 to correct the issue you mentioned, and also did some cleanup in the code. See [release notes](https://www.passbolt.com/release/notes#PapaWasARollingStone)

We have added a few test cases in our test suite to make sure that this bug doesn't happen again. [Here is a gist](https://gist.github.com/kevinmuller/cbdeb30e7122dd81fb052de7526bc28e) showing all the emails format supported and tested in our CI process, which we we got  from the CakePHP test suite. I think that it handles most of the cases, but suggestions of improvement are of course welcome if any.

Thanks again for flagging this.

Cheers,
